cli/http_util.rs

```diff
@@ -1,20 +1,29 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 use crate::deno_error;
 use crate::deno_error::DenoError;
+use crate::version;
 use deno::ErrBox;
 use futures::{future, Future};
 use reqwest;
+use reqwest::header::HeaderMap;
 use reqwest::header::CONTENT_TYPE;
 use reqwest::header::LOCATION;
+use reqwest::header::USER_AGENT;
 use reqwest::r#async::Client;
 use reqwest::RedirectPolicy;
 use url::Url;
 
 /// Create new instance of async reqwest::Client. This client supports
 /// proxies and doesn't follow redirects.
 pub fn get_client() -> Client {
+  let mut headers = HeaderMap::new();
+  headers.insert(
+    USER_AGENT,
+    format!("Deno/{}", version::DENO).parse().unwrap(),
+  );
```

Comment:
This prevents reqwest from sending its own user agent? did you check that there are not two user agent header lines?

Reply:
Yes, I dumped headers in echo server:
```
'user-agent: Deno/0.17.0\r\naccept: */*\r\naccept-encoding: gzip\r\nhost: localhost:4545\r\ncontent-length: 11\r\n'
```
There's only single `user-agent` header.